vmaas/vulnerabilities.go

```diff
@@ -181,10 +203,42 @@ func (d *ProcessedDefinition) evaluate(
 	}
 }
 
+// process repos into CPEs and ContentSets needed for vulnerability evaluation
+func (r *ProcessedRequest) processRepos(c *Cache) {
+	repoIDs, newerReleaseverRepoIDs, contentSetIDs := repos2IDs(c, r.OriginalRequest)
+	cpes := repos2cpes(c, repoIDs)
+	newerReleaseverCpes := repos2cpes(c, newerReleaseverRepoIDs)
+	r.Cpes = cpes
+	r.NewerReleaseverCpes = newerReleaseverCpes
+	r.ContentSets = contentSetIDs
+}
+
+func (r *ProcessedRequest) processProducts(c *Cache) []ProductsPackage {
+	productsPackages := make([]ProductsPackage, 0)
+	if r.OriginalRequest.UseCsaf {
+		for _, pkg := range r.Packages {
+			nameID := c.Packagename2ID[pkg.Nevra.Name]
+			products := cpes2products(c, r.Cpes, nameID, r.Updates.ModuleList, pkg)
```

Comment:
@jdobes should we use csaf with `newerReleaseverCpes` as we are doing it with oval for eus/aus/e4s repos?